src/containers/Balance/Balance.tsx

```diff
@@ -93,17 +94,17 @@ class Balance extends React.Component<Props, State> {
     const { value: amount } = event.target
     const myBalance = this.getMyBalance()
 
-    if (!myBalance || amount.includes('.')) {
+    if (!myBalance || !new RegExp('^[0-9]+$')) {
```

Comment:
this regexp is not applied to a string

Reply:
I am using Number or do you mean something else?

Reply:
Inside the input field.